Misc/NEWS.d/next/Core_and_Builtins/2024-10-01-17-31-32.gh-issue-124855.sdsv_H.rst

```diff
@@ -0,0 +1,2 @@
+Don't allow the JIT and perf support to be active at the same time. Patch by
```

Comment:
Maybe you should remove the author name, as it doesn't mean anything to the user.

Reply:
I have been doing this for 7 years to mark the contributions :)

Reply:
You can search for "patch by" in https://docs.python.org/3.13/whatsnew/changelog.html#changelog to see how common this is

Reply:
TBH I find it useful as by reading the changelog I have an idea who did what and I don't have to go through the endless PRs on GitHub.